src/views/partials/blogs/create.hbs

```diff
@@ -0,0 +1,20 @@
+<form action="/blogs/create" method="post" autocomplete="off" enctype="application/json">
+  <input name="id" type="hidden" value=" ">
```

Comment:
Id is always set to the same value., I think it's better to generate ids on the server-side.
And to use some automatic Id generator, like uuid, instead of counting up integers. Since it was done by @starmaerker so far, you can collaborate with him on this issue. 👍

Reply:
That's something I wanted to ask anyway, the set id was just meant as a placeholder. @starmaerker What do you think ybout this?

Reply:
MongoDB is generating already an unique id on the serverside. So we could replace id with _id in database.js(line 41) and posts.hbs(line 3). What do you think?

Reply:
Yeah, that's what I was thinking as well. @pheeria Does that work for you?

Reply:
@pheeria Sorry, busy with exams. Just to recap on what is left to do. So we don't use the timestamp and id that is automatically created but we add a unique uuid and date on server side for each created post?

![image](https://user-images.githubusercontent.com/34481945/54137118-54441880-441d-11e9-8308-7f37474a5e35.png)

Reply:
Yes, more or less. "id" should be auto-generated uuid :)